packages/server/src/config.ts

```diff
@@ -1,26 +1,66 @@
 import { env } from 'process';
 import { PERMISSIONS } from '../../lib/utils/constants';
 
+const JWT_PRIV_KEY = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why do we need defaults like this? Seems like it would be easy to accidentally forget to add the `JWT_PRIV_KEY` env var, and then the default and public one would be used.

Reply:
Good point, I have put them into a `.env` file that's not committed.